packages/@webex/plugin-meetings/test/unit/spec/meeting-info/meetinginfov2.js

```diff
@@ -826,18 +826,18 @@ describe('plugin-meetings', () => {
           statusCode: 200
         });
       });
-
       it('Make a request to /spaceInstant when conversationUrl with installed org ID', async () => {
         const {invitee} = setup();
-
+        webex.request = sinon.stub().resolves({
```

Comment:
What failure were we fixing here?

Reply:
As part of this [PR](https://github.com/webex/webex-js-sdk/commit/34f1f4659c936d5f46f6af99dd105c0fa4e30139#diff-73872a249ac2e9c1cee8cecfb0c1f1b2d63ca2ca0b01e01d563dcd5b3d285f91) we changed the code. From webex.internal.conversation we changed to webex.request but UTs were not changed.